lib/render/VolumeOSPRay.cpp

```diff
@@ -904,28 +934,29 @@ OSPVolume VolumeOSPRay::_loadVolumeUnstructured(const Grid *grid)
     //    Progress::Finish();
     //    printf("done\n");
 
-    //    int testCell = std::min(cellStarts.size()-1, (size_t)GetActiveParams()->GetValueLong("osp_test_cells", 0));
-    //    int testCell = 0;
-    //    if (testCell >= 0) {
-    //        int testCellNodes = cellTypes[testCell] == OSP_TETRAHEDRON ? 4 : 6;
-    //        printf("Test Cell[%i].nodes = %i\n", testCell, testCellNodes);
-    //        vec3 testCellCoords[testCellNodes];
-    //        for (int i = 0; i < testCellNodes; i++) {
-    //            int idx = cellIndices[cellStarts[testCell]+i];
-    //            testCellCoords[i] = coords[idx];
-    //            printf("\tCells[%i].vert[%i] = coords[%i] = (%f, %f, %f)\n", testCell, i, idx, coords[idx].x, coords[idx].y, coords[idx].z);
+    //        int testCell = std::min(cellStarts.size()-1, (size_t)maxCells-1);
```

Comment:
Can we put this diagnostic code in an ifdef per our coding conventions:

https://github.com/NCAR/VAPOR/wiki/Vapor-Coding-Convention#debugcomment

Reply:
Done